### PR TITLE
build Docker image on cron

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Remove a manual step from the release process.

### CODE REVIEW

**Comment by ndmitchell** [comment]
Can we update the release readme too.

File: azure-cron.yml [hunk 1]
```diff
@@ -118,3 +118,37 @@ jobs:
...
+      - checkout: self
+      - bash: |
+          set -euo pipefail
+
+          eval "$(dev-env/bin/dade-assist)"
+          echo $DOCKER_PASSWORD | docker login --username $DOCKER_LOGIN --password-stdin
+          RELEASES=$(curl https://api.github.com/repos/digital-asset/daml/releases -s | jq -r '. | map(select(.prerelease == false)) | map(.tag_name)[]')
+          DIR=$(pwd)
+          for version in $(echo $RELEASES | sed -e 's/ /\n/g'); do
+            EXISTS=$(curl 'https://hub.docker.com/v2/repositories/digitalasset/daml-sdk/tags/?page_size=10000' -s | jq -r '[.results[].name == "${version#v}"] | any')
```
Comment (ndmitchell) [inline azure-cron.yml]: Should the curl not be outside the loop

File: azure-cron.yml [hunk 2]
```diff
@@ -118,3 +118,37 @@ jobs:
...
+            if [ "$EXISTS" == "true" ]; then
+              echo "${version#v} already exists, skipping."
+            else
+              echo "Building version ${version#v}..."
+              cd $(mktemp -d)
+              echo "FROM openjdk:8u212-alpine" >> Dockerfile
+              echo "RUN apk add curl bash" >> Dockerfile
+              echo "RUN curl https://get.daml.com | sh -s ${version#v}" >> Dockerfile
+              docker build -t digitalasset/daml:${version#v} .
+              #docker push digitalasset/daml:${version#v}
```
Comment (ndmitchell) [inline azure-cron.yml]: Comment is just to try it out?

File: azure-cron.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply azure-cron.yml]: Right now the entire line is version-specific, but yes, the `curl` bit could be lifted out of the loop. Will do, thanks.

File: azure-cron.yml (hunk 2 above)
Comment (garyverhaegen-da) [reply azure-cron.yml]: Yes, as is the echo on the next line.

**Comment by garyverhaegen-da** [comment]
Hoisted `curl` outside the loop; removed manual Docker step from the README. Build [seems to be working](https://dev.azure.com/digitalasset/daml/_build/results?buildId=9320), including login with the Docker server and building an image for every version we have published so far.

**Pull request merged**